satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -1147,23 +1232,27 @@ datasets:
 
   qual:
     name: qual
+    long_name: Total Column Ozone Quality Index
+    standard_name: atmosphere_mass_content_of_ozone quality_flag
     key: '#1#totalOzone->totalOzoneQuality'
     resolution: 9001.209497451
-    standard_name: total_ozone_quality
     units: ""
     file_type: seviri_l2_bufr_toz
     coordinates:
        - longitude
        - latitude
     fill_value: 0
+    flag_values: [8, 9, 10, 11, 12]
+    flag_meanings: ['not processes', 'first guess', 'high cloud (not processed)', 'Success', 'bad retrieval']
```

Comment:
*'not processed'
*'successful retrieval'

Reply:
corrected